xcodecache/swiftpm_cache.go

```diff
@@ -0,0 +1,28 @@
+package cache
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/bitrise-io/go-steputils/cache"
+)
+
+// CollectPackagesCache marks the Swift Package Manager packages to be added the cache
```

Comment:
Comment should be a full sentence: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Comment should end with `.` char: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences